Quadriga/src/test/java/edu/asu/quadriga/validator/TextFileValidatorTest.java

```diff
@@ -0,0 +1,94 @@
+package edu.asu.quadriga.validator;
+
+import java.util.UUID;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.validation.BindException;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+
+import edu.asu.spring.quadriga.domain.impl.workspace.TextFile;
+import edu.asu.spring.quadriga.domain.workspace.ITextFile;
+import edu.asu.spring.quadriga.service.textfile.ITextFileManager;
+import edu.asu.spring.quadriga.service.textfile.impl.TextFileManager;
+import edu.asu.spring.quadriga.validator.AddTextValidator;
+import junit.framework.Assert;
+
+/**
+ * @author Nischal Samji
+ * 
+ * Unit Test for Validator for storing text file objects.
+ *
+ */
+public class TextFileValidatorTest {
+
+    @InjectMocks
+    private AddTextValidator txtValidator;
+
+    private ITextFile properTxtFile;
+    private ITextFile improperTxtFile;
+    private ITextFile emptyTxtFile;
+
+    @Before
+    public void init() {
+        
+        MockitoAnnotations.initMocks(this);
+
+        properTxtFile = new TextFile();
+        properTxtFile.setFileContent("Dummy File Content. This is Dummy");
+        properTxtFile.setFileName("testfile.txt");
+        properTxtFile.setProjectId(UUID.randomUUID().toString());
+        properTxtFile.setRefId(UUID.randomUUID().toString());
+        properTxtFile.setWorkspaceId(UUID.randomUUID().toString());
+
+        improperTxtFile = new TextFile();
+        improperTxtFile.setFileContent("");
+        improperTxtFile.setFileName(".testtext");
+        improperTxtFile.setProjectId(UUID.randomUUID().toString());
+        improperTxtFile.setRefId("");
+        improperTxtFile.setWorkspaceId(UUID.randomUUID().toString());
+
+        emptyTxtFile = new TextFile();
+        emptyTxtFile.setFileContent("");
+        emptyTxtFile.setFileName("");
+        emptyTxtFile.setWorkspaceId("");
+        emptyTxtFile.setProjectId("");
+        emptyTxtFile.setRefId("");
+
+    }
+
+    @Test
+    public void testProperTextFile() {
+        Errors errors = new BindException(properTxtFile, "pTxtFile");
+        ValidationUtils.invokeValidator(txtValidator, properTxtFile, errors);
+        Assert.assertFalse(errors.hasErrors());
+        Assert.assertNull(errors.getFieldError("fileName"));
+        Assert.assertNull(errors.getFieldError("fileContent"));
+        Assert.assertNull(errors.getFieldError("refId"));
```

Comment:
since you already checked that hasErrors is false, line 70-72 are not required. You're testing the Errors class here not the validator.